src/i18n/nl/index.ts

```diff
@@ -87,89 +87,89 @@ const nl = {
   },
   SETTINGS: {
     THEME: {
-      SYSTEM: '',
-      LIGHT: '',
-      DARK: '',
+      SYSTEM: 'Systeem',
+      LIGHT: 'Licht',
+      DARK: 'Donker',
     },
     PASSWORD: {
       POLICY: {
-        TITLE: '',
-        UPPERCASE_LETTER: '',
-        LOWERCASE_LETTER: '',
-        NUMBER: '',
-        CHARACTERS: '',
+        TITLE: 'Wachtwoord',
+        UPPERCASE_LETTER: 'Hoofdletters',
+        LOWERCASE_LETTER: 'Kleine letters',
+        NUMBER: 'Nummers',
+        CHARACTERS: 'Karakters',
       },
     },
     RESET_APP: {
-      TITLE: '',
-      DESCRIPTION: '',
-      CONFIRM: '',
-      CANCEL: '',
+      TITLE: 'App opnieuw instellen',
+      DESCRIPTION: 'Weet je zeker dat je de app opnieuw wilt instellen en alle gegevens wilt verwijderen? ',
```

Comment:
An empty space `' '` snuck in here at the end of the string :) 
`...verwijderen? '` should be `...verwijderen?'`